### PR TITLE
Feature/row limit

### PR DESCRIPTION
**Pull request by t-a-y-l-o-r**
Do you guys accept PRs?

I've been using your library for a while now on a project and have come across some less-than-ergonomic use cases.
Specifically when using pagination and limits.

This pr is a simple addition that gets the main behaviour in. It also includes some tests to hopefully encapsulate the appropriate behaviour.

Happy to discuss alternative solutions or quality concerns

### CODE REVIEW

**Comment by chdastolfo** [comment]
Hi @t-a-y-l-o-r  - we do, thanks for this! I'll take a look.

File: monday/tests/test_board_resource.py [hunk 1]
```diff
@@ -39,6 +39,14 @@ def test_get_boards_by_id_query(self):
     def test_get_board_items_query(self):
         query = get_board_items_query(board_id=self.board_id)
         self.assertIn(str(self.board_id), query)
+        items_line = f'items()'
```
Comment (chdastolfo) [inline monday/tests/test_board_resource.py:42]: Would prefer that this be its own test i.e. with and w/o pagination given that both are valid use cases, even if the second test is repetitive.

File: monday/query_joins.py [hunk 2]
```diff
@@ -289,12 +289,28 @@ def get_tags_query(tags):
 
 
 # BOARD RESOURCE QUERIES
-def get_board_items_query(board_id):
+def get_board_items_query(board_id: Union[str, int], limit: Optional[int] = None, page: Optional[int] = None) -> str:
+    def _gather_params(params) -> List[str]:
```
Comment (chdastolfo) [inline monday/query_joins.py:293]: While it's true that this method is only used here at the moment, I can see this being a useful util to reuse in the future with some tweaks (i.e. remove board specific logic - thoughts on passing in the list of invalid params, `board_id` and `_gather_params`, in here to make this generic?). I'd prefer that this method live outside here, in `utils.py`

**Review by chdastolfo — COMMENTED** [review]
Thanks for this! Could you also update the relevant method info in the README?

File: monday/query_joins.py (hunk 2 above)
Comment (t-a-y-l-o-r) [reply monday/query_joins.py:293]: ya, that's kinda why I like this style of nested functions. Makes it obvious what will need to be pulled out as common patterns start to appear. I can do another pass and get this into util

**Review by rhymiz — APPROVED** [review]

**Comment by rhymiz** [comment]
@chdastolfo you're good to merge

**Pull request merged**